packages/discord.js/src/structures/Role.js

```diff
@@ -217,7 +217,7 @@ class Role extends Base {
    *   .catch(console.error);
    */
   edit(data, reason) {
-    return this.guild.roles.edit(this, data, reason);
+    return this.guild.roles.edit(this, { ...data, reason });
   }
```

Comment:
Doesn't consistently take `reason` in `options`, it could be changed and do it pass-thru:
```js
  edit(data) {
    return this.guild.roles.edit(this, data);
  }
```